addons/website_form/views/snippets/s_website_form.xml

```diff
@@ -96,6 +92,15 @@
                 data-select-class="s_website_form_field_hidden"/>
         </div>
 
+        <div data-js="WebsiteFormSubmit" data-selector=".s_website_form_submit" data-exclude=".s_website_form_no_submit_options">
+            <we-select string="Button Position">
+                <we-button data-select-class="">Left</we-button>
+                <we-button data-select-class="text-center">Center</we-button>
+                <we-button data-select-class="text-right">Right</we-button>
+                <we-button data-select-class="s_website_form_label_aligned">Input Aligned</we-button>
```

Comment:
I push forced the branch to correct the apps in your commit message again, so fetch it before changing.
Here I see you introduced a new `s_website_form_label_aligned` class but you don't use it in your templates ? Left is the ugliest to me, label aligned should be the default. I would change the options here to select-class between:
- `text-left s_website_form_no_submit_label`
- `text-center s_website_form_no_submit_label`
- `text-right s_website_form_no_submit_label`
- `  `

I would force text-left since you don't know the inherited alignment of the form.

Also, there is bug with the input aligned option: it is not aligned because of a whitespace, can you also fix that?

Thanks.